### PR TITLE
fix(pie-docs): DSW-2327 content update changes in text input documentation 

### PR DESCRIPTION
**Pull request by raoufswe**
## Describe your changes (can list changeset entries if preferable)

[Fixed] - Content update changes in the text input documentation

- I also renamed some picture names to be more readable. ~

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] I have reviewed visual test updates properly before approving

### Reviewer 1 - @jamieomaguire 
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] If there are visual test updates, I have reviewed them

### Reviewer 2 - @ashleynolan 
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] If there are visual test updates, I have reviewed them

### CODE REVIEW

File: apps/pie-docs/src/assets/img/components/text-input/content-placeholder-dont.svg
Comment (jamieomaguire) [inline apps/pie-docs/src/assets/img/components/text-input/content-placeholder-dont.svg:1]: I don't want to block your PR but please can you feedback to design that this is a misuse of placeholders in the browser? Placeholder messages are not consistently read by screen readers and instructions on how to enter information seem more like candidates for assistive text.

It seems like a nit pick, but adding images like this just further enforces the idea of using placeholders for critical information.

Comment (jamieomaguire) [reply apps/pie-docs/src/assets/img/components/text-input/content-placeholder-dont.svg:1]: Reference: https://www.w3.org/WAI/tutorials/forms/instructions/#:~:text=If%20the%20placeholder%20text%20contains,not%20a%20replacement%20for%20labels.

Comment (raoufswe) [reply apps/pie-docs/src/assets/img/components/text-input/content-placeholder-dont.svg:1]: I am not sure if you noticed but this is a **don't** image so we are advocating for placeholder to not be like that

File: apps/pie-docs/src/components/text-input/overview/overview.md [hunk 1]
```diff
@@ -127,6 +127,11 @@ Use when text input requires clarification of the input format, such as a symbol
 Payment method is only available in trailing content.
 Use when a text input requires a card number to be entered, and reflect the card  entered by the user.
 
+{% notification {
+  type: "warning",
+  message: "This modifier only exists in design, not in development. The functionality is handled by pillar engineers' if required."
```
Comment (jamieomaguire) [inline apps/pie-docs/src/components/text-input/overview/overview.md:132]:
```suggestion
  message: "This modifier only exists in design, not in development. The functionality can be handled by consumers if required."
```

Comment (raoufswe) [reply apps/pie-docs/src/components/text-input/overview/overview.md:132]: i am gonna update the other references too 🤝

File: apps/pie-docs/src/components/text-input/overview/overview.md [hunk 2]
```diff
@@ -138,7 +143,7 @@ Use when a text input requires a card number to be entered, and reflect the card
 {% contentLayout %}
   {% contentItem %}
     <h4>Placeholder</h4>
-    <p>Placeholder text should not contain crucial information, and only displayed if beneficial to the user.</p>
+    <p>Placeholder text in the text input offers the user suggestions or examples for what to enter.</p>
```
Comment (jamieomaguire) [inline apps/pie-docs/src/components/text-input/overview/overview.md:146]: I think the previous message was a bit clearer in what not to do. Can I ask why we want to make this change?

Comment (raoufswe) [reply apps/pie-docs/src/components/text-input/overview/overview.md:146]: Discussed this with @tingting-lau and we decided to revert the change

**Review by jamieomaguire — DISMISSED** [review]

**Review by xander-marjoram — DISMISSED** [review]

**Comment by xander-marjoram** [comment]
A couple of images aren't loading, I think they might need updating:

https://pr1962-docs.pie.design/components/text-input/#card-number
https://pr1962-docs.pie.design/components/text-input/#payment-method-1

**Review by ashleynolan — APPROVED** [review]

**Review by maledr5 — APPROVED** [review]

**Review by jamieomaguire — APPROVED** [review]

**Pull request merged**